ArduCopter/Parameters.h

```diff
@@ -498,6 +499,7 @@ class Parameters {
     AC_HELI_PID             pid_rate_roll;
     AC_HELI_PID             pid_rate_pitch;
     AC_HELI_PID             pid_rate_yaw;
+    AC_PID                  pid_rotor_gov;
```

Comment:
Yeah, Randy just mentioned that last night.  This wasn't possible in the past, but it is now. But I'm not sure yet how to do it.  I can move it in before merge.